api/s2n.h

```diff
@@ -800,10 +825,12 @@ S2N_API extern int s2n_config_set_cert_chain_and_key_defaults(struct s2n_config
 S2N_API extern int s2n_config_set_verification_ca_location(struct s2n_config *config, const char *ca_pem_filename, const char *ca_dir);
 
 /**
- * Adds a PEM to the trust store. This will allocate memory, and load PEM into the
- * Trust Store. Note that the trust store will be initialized with the common locations
- * for the host operating system by default. To completely override those locations,
- * call s2n_config_wipe_trust_store before calling this function.
+ * Adds a PEM to the trust store. This will allocate memory and load a PEM into the trust store.
+ *
+ * If default system certificates should not be present in the trust store in addition to the
+ * certificates in `pem`, `config` should be created with `s2n_config_new_minimal()`, which will
+ * initialize the config with an empty trust store. To override a trust store that contains
+ * existing certificates, call `s2n_config_wipe_trust_store()` before calling this function.
```

Comment:
I'm not sure you're clear enough that the options are to either call s2n_config_new + s2n_config_wipe_trust_store OR call s2n_config_new_minimal.

Reply:
With the focus on performance, I changed this to just the previous recommendation of new + wipe.